packages/vx-xychart/src/components/series/LineSeries.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useContext, useCallback, useEffect } from 'react';
+import LinePath from '@vx/shape/lib/shapes/LinePath';
+import { ScaleConfig, ScaleConfigToD3Scale, ScaleInput } from '@vx/scale';
+import { AxisScaleOutput } from '@vx/axis';
+import DataContext from '../../context/DataContext';
+import isValidNumber from '../../typeguards/isValidNumber';
+
+type LineSeriesProps<
+  XScaleConfig extends ScaleConfig<AxisScaleOutput>,
```

Comment:
This seems like it could take 

```ts
  XScale extends AxisScale,
  YScale extends AxisScale,
```

instead of the configs, similar to the types below